study_time/courses/tests/fixtures.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from courses.tests import values
+from .factories import CourseFactory, EpisodeFactory
+
+
+@pytest.fixture
+def course_fixture():
+    return CourseFactory(id=values.COURSE_ID)
+
+
+@pytest.fixture
+def three_courses_fixture():
+    text_fields = {
+        "title": values.COURSE_TITLE,
+        "video_duration": values.COURSE_VIDEO_DURATION,
+        "language": values.COURSE_LANGUAGE
+        }
+    courses = [CourseFactory(id=id, **text_fields) for id in values.COURSES_IDS]
+    return courses
```

Comment:
Don't `three_courses_fixture` collide with `course_fixture` especially the id? I'd be safer if  `three_courses_fixture` used `course_fixture`.
```suggestion
@pytest.fixture
def three_courses_fixture(course_fixture):
    courses = [course_fixture]
    text_fields = {
        "title": values.COURSE_TITLE,
        "video_duration": values.COURSE_VIDEO_DURATION,
        "language": values.COURSE_LANGUAGE
        }
    courses += [CourseFactory(id=id, **text_fields) for id in values.COURSES_IDS[1:]
    return courses
```

Reply:
Another way is using the following set of the constants:
```
COURSE_ID = "..."  # is for course_fixture
THREE_COURSES_IDS = [...]  # exactly for three_courses_fixture
COURSES_IDS = [COURSE_ID] + THREE_COURSES_IDS
```